### DISCUSSION TITLE
Upgrade instances and local development to the latest stable of PostgREST (11.1.0)

### DISCUSSION THREAD

**Discussion by zachblume** [supabase/supabase#15508] [Feature Requests]
Nested/embedded relationships in PostgREST have received significant upgrades that I'd like to use.

Thinking of migrating from managed/pro to self-hosting to be able to have this dependency updated.

**Comment by iamgoddog** [comment]
I'm having the same issue. Waiting for the managed version to keep up.

**Reply by steve-chavez** [reply]
It's already on prod, you'll likely need to pause/unpause your instance. See https://supabase.com/blog/postgrest-11-1-release.

**Reply by icopp** [reply]
How does one pause/unpause a project with the current dashboard? The docs reference a pause button that doesn't exist.
